autoware_adapi_v1_msgs/vehicle/msg/VehicleTurnIndicator.msg

```diff
@@ -0,0 +1,7 @@
+# constants
+uint8 DISABLE = 1
+uint8 ENABLE_LEFT = 2
```

Comment:
I feel the ENABLE prefix is redundant.

Reply:
corrected in https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/ffc28266e7b4b930dcaccfbc83044af191c0b085